apps/trading/components/welcome-dialog/get-started.tsx

```diff
@@ -39,27 +95,40 @@ export const GetStarted = ({ lead }: Props) => {
     { 'mt-8': !lead }
   );
 
-  if (!pubKey && !isBrowserWalletInstalled()) {
+  if (getStartedNeeded) {
     return (
       <div className={wrapperClasses} data-testid="get-started-banner">
         {lead && <h2>{lead}</h2>}
         <h3 className="text-lg">{t('Get started')}</h3>
         <div>
-          <ul className="list-decimal list-inside">
-            <li>{t('Get a Vega wallet')}</li>
-            <li>{t('Connect')}</li>
-            <li>{t('Deposit funds')}</li>
-            <li>{t('Open a position')}</li>
+          <ul className="list-decimal list-inside" role="list">
+            <li>
+              {t('Get a Vega wallet')}{' '}
+              {currentStep > OnboardingStep.ONBOARDING_WALLET_STEP && (
+                <VegaIcon name={VegaIconNames.TICK} size={24} />
```

Comment:
Can we:

- render the check on the left instead of the number
- reduce icon size to 20

Reply:
done